quickwit/quickwit-search/src/lib.rs

```diff
@@ -199,15 +199,14 @@ pub async fn single_node_search(
     let metas = list_relevant_splits(index_uid, &search_request, metastore).await?;
     let split_metadata: Vec<SplitIdAndFooterOffsets> =
         metas.iter().map(extract_split_and_footer_offsets).collect();
-    validate_request(&*doc_mapper, &search_request)?;
+    let searcher_context = Arc::new(SearcherContext::new(SearcherConfig::default()));
```

Comment:
```suggestion
    let searcher_context = SearcherContext::new(SearcherConfig::default());
```

Reply:
fix(This `searcher_context` here has been reverted to its previous state)
https://github.com/quickwit-oss/quickwit/pull/3566/commits/9d02896663ee4181c52422baf6c61cf55b4b29f5